src/ApiCore/GapicClientTrait.php

```diff
@@ -162,6 +172,26 @@ private function buildClientOptions(array $options)
             'rest' => [],
         ];
 
+        if (isset($options['gcpApiConfigPath'])
+                && file_exists($options['gcpApiConfigPath'])
+                && isset($options['serviceAddress'])) {
+            $grpcGcpConfig = initGrpcGcpConfig(
```

Comment:
`$this->initGrpcGcpConfig`?

Reply:
Thx. Seems like this line is not being covered in tests, I'll double check

Reply:
Done.